demo/app-root.ts

```diff
@@ -1,21 +1,134 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+/* eslint-disable no-restricted-globals */
+import {
+  html,
+  css,
+  LitElement,
+  customElement,
+  property,
+  query,
+} from 'lit-element';
+import '../src/item-inspector/item-inspector';
+import {
+  MetadataResponse,
+  SearchService,
+} from '@internetarchive/search-service';
+import '../src/item-navigator';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @property({ type: Object }) itemMD: MetadataResponse | undefined = undefined;
+
+  @property({ type: Object }) bookManifest = {};
+
+  @property({ type: String }) encodedManifest = '';
+
+  @query('item-navigator') private itemNav!: any;
+
+  firstUpdated() {
+    this.fetchItemMD();
+    // this.fetchDemoBook();
+  }
+
+  /**
+   * @inheritdoc
+   */
+  updated(changed: any) {
+    if (changed.has('itemMD')) {
+      this.fullscreenCheck();
+    }
+  }
+
+  fullscreenCheck() {
+    if (this.showFullscreen && this.itemNav) {
+      this.itemNav.viewportInFullscreen = true;
+    }
+  }
+
+  async fetchItemMD() {
+    const searchService = SearchService.default;
+
+    const mdResponse = await searchService.fetchMetadata('ux-team-books');
+
+    if (mdResponse.error) {
+      console.log('MD Fetch error: ', mdResponse.error);
+      (window as any).confirm(
+        'There was an error fetching response, please check dev console'
+      );
+      return;
+    }
+
+    this.itemMD = mdResponse.success;
+  }
+
+  get urlParams(): URLSearchParams {
+    return new URLSearchParams(location.search.slice(1));
+  }
+
+  get showFullscreen(): boolean {
+    return this.urlParams.get('view') === 'theater';
+  }
+
+  get renderMD() {
+    const x = [];
```

Comment:
SUGGESTION: Choose a more descriptive variable name. Even here, one can strive for legibility. :)